### ISSUE TITLE
When you click on the strength value of the powerora node, the numeric pop-up does not appear.

### ISSUE THREAD

**Issue by ytoaa** [rgthree/rgthree-comfy#385]
I'm using the latest version of comfyui as of November 29
Left and right adjustments change the value, but you cannot modify the number directly.

**Comment by LukeG89** [comment]
Same issue as this: https://github.com/rgthree/rgthree-comfy/issues/375#issuecomment-2501119008

It's fixed in frontend 1.5.0

**Comment by rgthree** [comment]
Yup, ComfyOrg accidentally broke this and they've fixed it in `ComfyUI_frontend v1.5.0`. See this comment https://github.com/rgthree/rgthree-comfy/issues/375#issuecomment-2507584179 for more info. Thx
